{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "7b7e9dbe_ac98b6e5",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1001921
      },
      "writtenOn": "2021-04-19T18:11:07Z",
      "side": 1,
      "message": "TBR",
      "revId": "0011e7563bb9547167a660dd3ce24eb1aee5d5e1",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "900f8ff9_173b1fec",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1392020
      },
      "writtenOn": "2021-04-19T18:19:02Z",
      "side": 1,
      "message": "Thanks. Maybe the failure is sporadically that accidentally passed it when I added the test? But suppress it is the thing here.",
      "parentUuid": "7b7e9dbe_ac98b6e5",
      "revId": "0011e7563bb9547167a660dd3ce24eb1aee5d5e1",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}